.travis.yml

```diff
@@ -1,7 +1,6 @@
-language: objective-c
-
-osx_image: xcode8.3
```

Comment:
xcode 8.3 is used to ensure 10.9 compatibilty

Reply:
Carthage was failing on 8.3, I will try by upgrading carthage in the `before_install`

Reply:
Carthage is not able to build the dependency on that image. Seems to be a signing issue that has been resolved on newer version..

Reply:
Can you elaborate? Any way to fix this?

Another idea might be to build up a matrix with macOS using 8.3 and the all the iOS derivates using a newer toolchain?

Reply:
Well I didnt dig into carthage release notes, but I often get that error when code signing fails on CI